src/gc/gc.cpp

```diff
@@ -5157,29 +5157,27 @@ void set_thread_affinity_mask_for_heap(int heap_number, GCThreadAffinity* affini
             {
                 if (bit_number == heap_number)
                 {
-                    dprintf (3, ("Using processor %d for heap %d\n", proc_number, heap_number));
+                    printf ("Using processor %d for heap %d\n", proc_number, heap_number);
```

Comment:
yeah, this is not ready for review. I just wanted to put it through the non windows builds to make sure it compiles ok. I'll ask for a CR when it's ready.